packages/docs/src/components/page-configs/ui-elements/va-data-table/api-options.ts

```diff
@@ -0,0 +1,36 @@
+import { defineManualApi } from '../../../DocsApi/ManualApiOptions'
+
+export default defineManualApi({
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/135526956-064525d6-ed92-4025-b2b6-9e2547ff332b.png)

Can we use less bright color? :D

Reply:
we use `--selected-color` calculated by `getFocusColor` (from our color-config functions) based on passed by user `selectedColor` prop. And the `focus-color` in this case is  quite a bit brighter than the `hover-color`. I think its a good decision.
Maybe, in this example I use less bright color in `selectedColor` prop.